src/contact/ContactForm.js

```diff
@@ -46,21 +47,29 @@ class ContactForm extends React.Component {
 
   render() {
     return (
-      <div>
-      <h1>Contact Us!</h1>
-        <form onSubmit={this.submitHandler}>
-          <div>
-          <label htmlFor="firstName">First Name</label>
-          <input type="text" name="firstName" id="firstName" value={this.state.firstName} onChange={this.handleFormChange}/>
-          <label htmlFor="lastName">Last Name</label>
-          <input type="text" name="lastName" id="lastName" value={this.state.lastName} onChange={this.handleFormChange} />
-          <label htmlFor="email">Email</label>
-          <input type="text" name="email" id="email" value={this.state.email} onChange={this.handleFormChange}/>
-          <label htmlFor="message">Message</label>
-          <textarea name="message" id="message" value={this.state.message} onChange={this.handleFormChange} cols="30" rows="10" />
-          <input type="submit" id="submitButton" onClick={this.submitHandler} value="Submit"/>
-          </div>
-        </form>
+      <div className="form-component-container">
+        <h1>Contact Us!</h1>
+        <Form className="contact-form" onSubmit={this.submitHandler}>
+          <Form.Group controlId="formFirstName">
+            <Form.Label>First Name</Form.Label>
+            <Form.Control type="text" name="firstName" onChange={this.handleFormChange} />
+          </Form.Group>
+          <Form.Group controlId="formLastName">
+            <Form.Label>Last Name</Form.Label>
+            <Form.Control type="text" name="lastName" onChange={this.handleFormChange} />
+          </Form.Group>
+          <Form.Group controlId="email">
```

Comment:
Keep naming style consistent
```suggestion
          <Form.Group controlId="formEmail">
```